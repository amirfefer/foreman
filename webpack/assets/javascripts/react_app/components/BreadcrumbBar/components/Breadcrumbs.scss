.breadcrumbs-pf-title .active span {
  display: inline-flex;
  max-width: 50%;
}
