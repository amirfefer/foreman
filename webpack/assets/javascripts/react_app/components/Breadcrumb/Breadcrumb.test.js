import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import React from 'react';
import { mockedBreadcrumbsMenu, headerTitle } from './Breadcrumbs.fixtures';
import ForemanBreadcrumb from './';

jest.unmock('./');

describe('render breadcrumbs', () => {
  it('renders breadcrumbs menu', () => {
    const wrapper = shallow(<ForemanBreadcrumb data={mockedBreadcrumbsMenu}/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('renders h1 title', () => {
    const wrapper = shallow(<ForemanBreadcrumb data={headerTitle}/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
