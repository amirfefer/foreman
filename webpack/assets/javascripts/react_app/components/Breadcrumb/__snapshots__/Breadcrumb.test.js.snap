// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render breadcrumbs renders breadcrumbs menu 1`] = `
<div>
  <Breadcrumb
    title={true}
  >
    <BreadcrumbItem
      active={false}
      dangerouslySetInnerHTML={
        Object {
          "__html": "root",
        }
      }
      href="/some-url"
      key="0"
    />
    <BreadcrumbItem
      active={true}
      dangerouslySetInnerHTML={
        Object {
          "__html": "child",
        }
      }
      key="1"
    />
  </Breadcrumb>
  <hr
    className="breadcrumb-line"
  />
</div>
`;

exports[`render breadcrumbs renders h1 title 1`] = `
<div
  className="form-group"
>
  <h1>
    title
  </h1>
</div>
`;
